src/modules/offline-page/index.ts

```diff
@@ -25,8 +25,8 @@ export type OfflinePageOptions = {
 
 export class OfflinePageAmpSwModule implements AmpSwModule {
   async init(url: string, assets: Array<string>) {
-    const publisherCache = await caches.open(documentCache);
-    const assetsCache = await caches.open(assetCache);
+    const publisherCache = await caches.open(AMP_PUBLISHER_CACHE_NAME);
+    const assetsCache = await caches.open(ASSET_CACHE);
```

Comment:
Looks like a good place to use `Promise.all` or similar.

Reply:
Done